version.php

```diff
@@ -29,8 +29,7 @@
 
 defined('MOODLE_INTERNAL') || die();
 
-$plugin->version = 2024022600;
+$plugin->version = 2024043000;
 $plugin->requires = 2023041800;
 $plugin->component = 'repository_ocis';
-$plugin->maturity = MATURITY_RC;
```

Comment:
should be set to stable `$plugin->maturity = MATURITY_STABLE;`
see https://moodledev.io/docs/4.4/apis/commonfiles/version.php#maturity